packages/viewer/src/properties.ts

```diff
@@ -301,4 +308,13 @@ export class Properties {
     this.config.backendConfig.wiriseditormathmlattribute = wiriseditormathmlattribute;
     this.render();
   }
+
+  get blacklist(): string | null {
+    return this.config.blacklist || defaultValues.blacklist;
```

Comment:
```suggestion
    return this.config.blacklist ?? defaultValues.blacklist;
```

Sugerencia de tu querido sonarlint!!!!